README.md

```diff
@@ -1,239 +1,108 @@
-</p>
-
-<p align="center">
-  <a href="http://www.overture.bio/products/ego" target="_blank"><img alt="General Availability" title="General Availability" src="http://www.overture.bio/img/progress-horizontal-GA.svg" width="320" /></a>
-</p>
-
-[![BrowserStack Status](https://www.browserstack.com/automate/badge.svg?badge_key=U3dLZnRFNWI2MWNFY2NGcXVtVTB3WDcyU2dPVjlVeEFYUEdxUnpYZlhrUT0tLTFzY0taYTA0MVFEa3ErNkRZdTBRWVE9PQ==--690f89a41a0eedf7b4975bd7df2eac162e04e775% )](https://www.browserstack.com/automate/public-build/U3dLZnRFNWI2MWNFY2NGcXVtVTB3WDcyU2dPVjlVeEFYUEdxUnpYZlhrUT0tLTFzY0taYTA0MVFEa3ErNkRZdTBRWVE9PQ==--690f89a41a0eedf7b4975bd7df2eac162e04e775%)
-[![Slack](http://slack.overture.bio/badge.svg)](http://slack.overture.bio)
-
-## Table of Contents
-- [Documentation](#documentation)
-- [Introduction](#introduction)
-- [Features](#features)
-- [Tech Stack](#tech-stack)
-- [Quick Start](#quick-start)
-- [Development Install](#development-install)
-  - [Step 1 - Setup Database](#step-1---setup-database)
-  - [Step 2 - Run](#step-2---run)
-- [Tech Specifications](#tech-specification)
-- [Usage](#usage)
-- [Shoutouts](#shoutouts)
-  - [Browserstack](#browserstack)
+# Ego - Authentication and Authorization Microservice
 
-## Documentation
-
-Explore documentation with the Ego [Read the Docs](https://ego.readthedocs.io/en/latest/).
-
-## Introduction
-
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-
-Authorization Service built to provide Single Sign On for various microservices in an application. EGO works with Identity Providers such as Google, Facebook to provide social logins in the application. EGO provides stateless authorization using [JWT (JSON Web Tokens)](https://jwt.io/) and can scale very well to a large number of users.
+[<img hspace="5" src="https://img.shields.io/badge/chat-on--slack-blue?style=for-the-badge">](http://slack.overture.bio)
+[<img hspace="5" src="https://img.shields.io/badge/License-gpl--v3.0-blue?style=for-the-badge">](https://github.com/overture-stack/ego/blob/develop/LICENSE)
+[<img hspace="5" src="https://img.shields.io/badge/Code%20of%20Conduct-2.1-blue?style=for-the-badge">](code_of_conduct.md)
 
-Interactive documentation of the API is provided using Swagger UI.
+<div>
+<img align="right" width="66vw" vspace="5" src="icon-ego.png" alt="ego-logo" hspace="30"/>
+</div>
 
-When run locally this can be found at: [http://localhost:8081/swagger-ui.html](http://localhost:8081/swagger-ui.html)
+Biomedical data requires secure protocols for authenticating users and authorizing the information and applications those users can access. [Ego](https://www.overture.bio/products/ego/) addresses this by facilitating user registration and providing a secure permission management system. 
+An [Ego UI tool](https://github.com/overture-stack/ego-ui) was also developed to make these services accessible to all collaborators.
```

Comment:
don't need "tool", just `Ego UI` should be fine. Rephrasing like:
> Ego UI is also available to make the Ego service accessible for system administrators.

Reply:
removed!